src/comms.cpp

```diff
@@ -20,67 +21,272 @@
 struct PacketAction {
     Packet pkt;
     bool inc_counter;
+    uint32_t delay;
 };
 
 Queue_t pkt_q;
 SoftwareSerial sw_serial;
+extern long unsigned int led_on_time;
+
 extern struct GarageDoor garage_door;
 
-SecPlus2Reader reader;
+uint8_t gdoSecurityType;
+
+/******************************* SECURITY 2.0 *********************************/
 
+SecPlus2Reader reader;
 uint32_t id_code = 0;
 uint32_t rolling_code = 0;
 
-extern long unsigned int led_on_time;
+/******************************* SECURITY 1.0 *********************************/
+
+static const uint8_t RX_LENGTH = 2;
+typedef uint8_t RxPacket[RX_LENGTH * 4];
+unsigned long last_rx;
+unsigned long last_tx;
+
+bool wallplateBooting= false;
+bool wallPanelDetected = false;
+DoorState doorState = DoorState::Unknown;
+uint8_t lightState;
+uint8_t lockState;
+
+//byte secplus1States[19] = {0x35,0x35,0x35,0x35,0x33,0x33,0x53,0x53,0x38,0x3A,0x3A,0x3A,0x39,0x38,0x3A, 0x38,0x3A,0x39,0x3A};
```

Comment:
Why the commented out States.  Either remove or add some comments around what this one is, or maybe an ifdef.  Not sure what makes sense here.